### ISSUE TITLE
About 'unused-variable' in LTR390.cpp

### ISSUE THREAD

**Issue by tfuruha** [levkovigor/LTR390#8]
Hi.
If there are no other problems, could you please remove the return value `byte ack` of `writeRegister()` on line 53 of LTR390.cpp?
I presume that this is due to the Arduino IDE ESP32 Core, but since "-Werror=unused-variable" is enabled, a compiler error occurs as below.
```
c:\Users\_____\Documents\Arduino\libraries\LTR390\src\LTR390.cpp:53:8: error: unused variable 'ack' [-Werror=unused-variable]
   byte ack = writeRegister(LTR390_MAIN_CTRL, _r);
        ^~~
cc1plus.exe: some warnings being treated as errors
```

**Comment by levkovigor** [comment]
@tfuruha 
fixed in release 1.0.9

**Comment by tfuruha** [comment]
Thanks.
